HW5/DubovskyiIhor/Task2.py

```diff
@@ -0,0 +1,6 @@
+login=input('Input your login: ')
```

Comment:
user_name=input('Hello, please input your Log in:')

while user_name!='First':
    # print('Error: wrong username, please try one more time.')
    user_name=input('Error: wrong username, please try one more time. Username:')
    # print(user_name)

else:
    print('Greeting. Access granted!!!', user_name)